packages/kit/package.json

```diff
@@ -25,7 +25,6 @@
 		"kleur": "^4.1.4",
 		"locate-character": "^2.0.5",
 		"marked": "^2.0.3",
-		"micromatch": "^4.0.4",
```

Comment:
`@types/micromatch` can be removed, too.

Reply:
Nice catch!